docs/book/src/migration/v2vsv3.md

```diff
@@ -2,21 +2,71 @@
 
 This document covers all breaking changes when migrating from v2 to v3.
 
-The details of all changes (breaking or otherwise) can be found in [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases) release notes.
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+v3 projects use go modules and at least Go 1.15; Dep is no longer supported.
```

Comment:
```suggestion
v3 projects use Go modules and request Go 1.15 or newer. Dep is no longer supported for dependency management.
```

Reply:
We cannot ensure newer such as 1.16. we can 1.15+. I will update that.